qiskit_ibm_runtime/runtime_job.py

```diff
@@ -655,3 +656,19 @@ def tags(self) -> List:
             Tags assigned to the job that can be used for filtering.
         """
         return self._tags
+
+    @property
+    def usage_estimation(self) -> Dict[str, Any]:
+        """Return the usage estimation infromation for this job.
+
+        Returns:
+            ``quantum_seconds`` which is the estimated running
+            time of the job in seconds.
```

Comment:
Perhaps we can be more explicit here, since this is different from the unit used in `max_execution_time`. Perhaps something like

```suggestion
            ``quantum_seconds`` which is the estimated quantum time 
            of the job in seconds. Quantum time represents the time that 
            the QPU complex is occupied exclusively by the job.
```

Reply:
sure, will copy these changes to qiskit-ibm-provider as well